test/util/card-link.js

```diff
@@ -0,0 +1,11 @@
+class CardLink
+{
+  constructor(key, type, linkType)
+  {
+    this.key = key;
+    this.type = type;
+    this.linkType = linkType;
+  }
+}
+
+export default CardLink;
```

Comment:
This works just as well as `export default class CardLink {` only with extra (unnecessary) verbosity.